service/publishing.gradle

```diff
@@ -1 +1,45 @@
-// TODO! Maybe get started by copy/pasting from data-catalog?
+import java.time.ZonedDateTime
+
+// Download and extract the Cloud Profiler Java Agent
+ext {
+    // where to place the Cloud Profiler agent in the container
+    cloudProfilerLocation = "/opt/cprof"
+
+    // location for jib extras, including the Java agent
+    jibExtraDirectory = "${buildDir}/jib-agents"
+}
+task downloadProfilerAgent(type: Download) {
+    // where to download the Cloud Profiler agent https://cloud.google.com/profiler/docs/profiling-java
+    src "https://storage.googleapis.com/cloud-profiler/java/latest/profiler_java_agent.tar.gz"
+    dest "${buildDir}/cprof_java_agent_gce.tar.gz"
+}
+task extractProfilerAgent(dependsOn: downloadProfilerAgent, type: Copy) {
+    from tarTree(downloadProfilerAgent.dest)
+    into "${jibExtraDirectory}/${cloudProfilerLocation}"
+}
```

Comment:
The other approach here is to use a `Dockerfile` to install the profiler agent. I copied this approach from another project, and it works, but I've thought that the dockerfile version might be a bit cleaner. The main difference is that the agent archive is downloaded and installed by docker, not by gradle. An example is https://github.com/DataBiosphere/terra-external-credentials-manager/blob/dev/service/Dockerfile

Reply:
At first glance, I like the `jib` option more, but it feels like it would be relatively easy to switch between them if some compelling reason came up